clusterproperty/hack/boilerplate.go.txt

```diff
@@ -0,0 +1,15 @@
+/*
+Copyright 2022.
```

Comment:
```suggestion
Copyright 2022 The Kubernetes Authors.
```

Copyrights should attribute the Kubernetes project 👍 
One suggestion though - it might be worth bringing over the verify-boilerplate scripts from k/k or one of the other repos. They will account for the year if any files are added in 2023 and so forth :)